### PR TITLE
wordpress: set banner via php env var

### PR DESCRIPTION
**Pull request by timmywil**
See companion PR at https://github.com/jquery/jquery-wp-content/pull/468

### CODE REVIEW

File: modules/profile/templates/wordpress/docs/jquery-config.php.erb [hunk 1]
```diff
@@ -14,3 +14,10 @@ define( 'JQUERY_STAGING_FORMAT', 'stage.%s' );
 if ( defined( 'XMLRPC_REQUEST' ) ) {
   @ini_set( 'memory_limit', '512M' );
 }
+
+<%- if @live_site == 'jquery.com' || @live_site == 'api.jquery.com' -%>
+define(
+  'JQUERY_BANNER',
+  'jQuery 4.0 is currently in beta. Once released, jQuery 3.x will only receive critical updates. Learn more about our <a href="https://jquery.com/support/">Version Support</a>.'
```
Comment (Krinkle) [inline modules/profile/templates/wordpress/docs/jquery-config.php.erb]: It feels slightly off to me, emotionally, to describe jQuery 3 as "only" this or that when it is still in the future. It sounds like jQuery 3 is deprecated, or soon to be deprecated, but there's nothing I can do about it?

If I was on an old 1.x, 2.x, or older 3.x version, I would not feel addressed by this message, either. Which seems like the core group we want to target, also in light of the ecosystem focus from the past year on security and outdated versions. 

Should we encourage users to be "prepared for jQuery 4.0" by "upgrading to the latest 3.x" now if you're on an older version?

Current:
> jQuery 4.0 is currently in beta. Once released, jQuery 3.x will only receive critical updates. Learn more about our [Version Support](https://jquery.com/support/).

Perhaps something like:
> jQuery 4.0 is [coming soon](https://blog.jquery.com/)! Prepare by upgrading to the latest jQuery 3.x release. Learn more at [Version Support](https://jquery.com/support/).

Comment (timmywil) [reply modules/profile/templates/wordpress/docs/jquery-config.php.erb]: I like that a lot

Comment (timmywil) [reply modules/profile/templates/wordpress/docs/jquery-config.php.erb]: I tweaked the links a little, but essentially used your message instead.

**Review by Krinkle — APPROVED** [review]

File: modules/profile/templates/wordpress/docs/jquery-config.php.erb [hunk 2]
```diff
@@ -14,3 +14,10 @@ define( 'JQUERY_STAGING_FORMAT', 'stage.%s' );
 if ( defined( 'XMLRPC_REQUEST' ) ) {
   @ini_set( 'memory_limit', '512M' );
 }
+
+<%- if @live_site == 'jquery.com' || @live_site == 'api.jquery.com' -%>
+define(
+  'JQUERY_BANNER',
+  'jQuery 4.0 is coming soon! Prepare by <a href="https://jquery.com/download/">upgrading</a> to the latest jQuery 3.x release. Learn more about our <a href="https://jquery.com/support/">version support</a>.'
```
Comment (mgol) [inline modules/profile/templates/wordpress/docs/jquery-config.php.erb:21]: The best way to prepare would be to update to the latest jQuery 3.x, enable Migrate for a while and make sure no warnings are being issued.

One problem with this approach, though, is adding Migrate will also reintroduce jQuery 1.9 behavior in many APIs which is undesirable. I have an issue to tackle this problem in Migrate: https://github.com/jquery/jquery-migrate/issues/472.

For now, this is good, I think.

**Review by mgol — APPROVED** [review]

**Pull request merged**